### PR TITLE
[SIG 4555] difference bedankt pages.

### PR DESCRIPTION
**Pull request by OscarBakker**
## Signalen

Before opening a pull request, please ensure:
https://datapunt.atlassian.net/browse/SIG-4555?atlOrigin=eyJpIjoiZDg5OTA5MWZmNDlhNGY3Mjg5YmEwNThjOWE4NGY1ZjEiLCJwIjoiaiJ9

**voor tester**
http://localhost:3001/kto/nee/69eed4cb-00c0-a3b9-cd1c-3368ba5de26fbijvoorbeeld
- verander de nee met de ja voor de andere tekst

- [x] Double-check your branch is based on `develop` and targets `develop`
- [x] Pull request has tests (we are going for 100% coverage!)
- [x] Code is well-commented, linted and follows project conventions
- [x] Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: src/signals/incident/containers/KtoContainer/index.js [hunk 1]
```diff
@@ -108,17 +122,11 @@ export const KtoContainer = () => {
   const parseResponse = useCallback(async () => {
     let payload = ''
 
-    try {
-      const { detail } = errorCheck
```
Comment (OscarBakker) [inline src/signals/incident/containers/KtoContainer/index.js:112]: try catch not needed!

Comment (kiruys) [reply src/signals/incident/containers/KtoContainer/index.js:112]: 👍

**Review by kiruys — APPROVED** [review]
Looks good! One minor thing, can you do something about the warning in Selector.test.tsx? I have fixed the other one.

**Review by kiruys — CHANGES_REQUESTED** [review]
checkte de branch nog even uit en toen zag ik toch nog een paar dingetjes:

- in het ja formulier staat nog de vraag of er contact opgenomen mag worden. 
- de balk onder de header klopt ook niet helemaal (de balk is te smal).

**Comment by kiruys** [comment]
And the name of the feature flag is `REPORTER_MAIL_HANDLED_NEGATIVE_CONTACT_ENABLED`

**Comment by OscarBakker** [comment]
> And the name of the feature flag is `REPORTER_MAIL_HANDLED_NEGATIVE_CONTACT_ENABLED`

- [ ] toevoegen en testen

**Review by kiruys — APPROVED** [review]

**Pull request merged**